.github/workflows/eval.yml

```diff
@@ -267,3 +267,13 @@ jobs:
           GH_TOKEN: ${{ github.token }}
           REPOSITORY: ${{ github.repository }}
           NUMBER: ${{ github.event.number }}
+
+      - name: Adding comments
```

Comment:
I am not so sure about comments here. This will be posted everytime someone pushes to a pull request and a new eval starts. This sounds noisy to me. We mostly care for these changes if they are mass rebuilds and for that we have labels.

Reply:
How about `gh pr comment "$NUMBER" --edit-last --body-file comparison/step-summary.md`?

It should post only a comment, and update it on every push.